roles/ceph-common/defaults/main.yml

```diff
@@ -302,6 +306,15 @@ radosgw_keystone_token_cache_size: 10000
 radosgw_keystone_revocation_internal: 900
 radosgw_s3_auth_use_keystone: "true"
 radosgw_nss_db_path: /var/lib/ceph/radosgw/ceph-radosgw.{{ ansible_hostname }}/nss
+# Settings for the RGW usage logging described at http://docs.ceph.com/docs/jewel/man/8/radosgw/#usage-logging
+radosgw_usage_log: false
+radosgw_usage_log_tick_interval: 30
```

Comment:
We could potentially fix that by introducing a really disruptive change. There is no need for a rgw.$(hostname) section, we can hardcode the name of the gateway with something like `rados.gateway`. IIRC we actually used to do that...
However this is just so disruptive for current user, and I don't know how we'd maintain a backward compatibility...